src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+
+<body>
+
+	<header>
+		<nav class="navbar">
+			<div class="logo-container">
+				<a href="#" class="logo">Netflix</a>
+			</div>
+			<ul class="primary-navigation">
+				<li class="navigation-tab"><a href="#">Home</a></li>
+				<li class="navigation-tab"><a href="#">TV Shows</a></li>
+				<li class="navigation-tab"><a href="#">Movies</a></li>
+			</ul>
+		</nav>
```

Comment:
On the navigation, kindly create a counter that shows the total number shows that are displayed on the homepage as it is on the [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md).

![image](https://user-images.githubusercontent.com/38088818/212419578-d3c528e3-1cd8-46ae-b659-5181b3db0fac.png)

Your navigation:

![image](https://user-images.githubusercontent.com/38088818/212420385-21a1047c-65d7-4341-90f6-db95f15c49be.png)

Reply:
Fixed tv shows counter UI display issues and menu items gap.